packages/-test-infra/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@ember-data/-test-infra",
+  "version": "3.13.0-canary.0",
+  "description": "The default blueprint for ember-data private packages.",
+  "keywords": [],
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+  },
+  "dependencies": {
+    "ember-cli-blueprint-test-helpers": "^0.19.1"
```

Comment:
I suspect more should be here (like mocha)

Reply:
mocha isn't 1`require`d by any of the files.